### PR TITLE
updated the yarn-audit-known-issues file

### PR DESCRIPTION
**Pull request by leepavlouhmcts**
### JIRA link (if applicable) ###
no jira

### Change description ###
a temporary fix, to ignore the audit errors

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by rayliangatos — COMMENTED** [review]
1. we probably don't need to update yarn-audit-known-issues
2. the whole yarn.lock is modified looks like abnormal

File: package.json [hunk 1]
```diff
@@ -205,6 +204,8 @@
     "**/**/ini": "^1.3.6",
     "**/**/elliptic": "^6.5.4",
     "**/**/y18n": "^3.2.2",
-    "**/**/jose": "1.28.1"
+    "**/**/jose": "1.28.1",
+    "**/**/**/ssri": "^8.0.1",
```
Comment (Daniel-Lam) [inline package.json:208]: @leepavlouhmcts - same question as on the other PR, regarding the number of `**/`'s

File: package.json [hunk 2]
```diff
@@ -205,6 +204,8 @@
...
+    "**/**/**/**/xmlhttprequest-ssl": "^1.6.2"
```
Comment (Daniel-Lam) [inline package.json:209]: As above

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@leepavlouhmcts - just two comments that need confirmation.

File: package.json (hunk 2 above)
Comment (leepavlouhmcts) [reply package.json]: replaced **/**./**/** to **/**/

File: package.json (hunk 1 above)
Comment (leepavlouhmcts) [reply package.json]: replaced **/**./**/** to **/**/

File: package.json [hunk 3]
```diff
@@ -161,15 +161,14 @@
     "jasmine-core": "^2.99.1",
     "jasmine-marbles": "^0.4.1",
     "jasmine-spec-reporter": "~4.2.1",
-    "json-server": "^0.14.2",
+    "json-server": "^0.16.3",
```
Comment (Daniel-Lam) [inline package.json:164]: @leepavlouhmcts - just querying whether this needs to go up to `0.16.3` or can be left at `0.14.2`?

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (leepavlouhmcts) [reply package.json:164]: if i put it back it runs but i get 2 additiional vulnerabilities, 1 high and one moderate which i have to add to resolutions

Comment (Daniel-Lam) [reply package.json:164]: Resolving as OK, following confirmation from @leepavlouhmcts.

**Review by Daniel-Lam — APPROVED** [review]

**Pull request closed without merging**